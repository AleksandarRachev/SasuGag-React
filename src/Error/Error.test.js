import React from 'react';
import Error from './Error';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Error', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Error />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  // it('should call function handleClose', () => {
  //     const wrapper = shallow(<Error />);
  //     const button = wrapper.find('.btn-msg-close');
  //     let spyHandleClose = jest.spyOn(Error.prototype, 'handleClose');
  //     button.simulate('click');
  //     expect(spyHandleClose).toHaveBeenCalled();

  //     // let spySetOpen = jest.spyOn();
  //     // expect(spyScrollTo).toHaveBeenCalled();
  //   });
});
