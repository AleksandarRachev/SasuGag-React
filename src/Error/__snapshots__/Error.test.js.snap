// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render without crashing 1`] = `
<WithStyles(ForwardRef(Snackbar))
  anchorOrigin={
    Object {
      "horizontal": "center",
      "vertical": "top",
    }
  }
  autoHideDuration={10000}
  className="position"
  onClose={[Function]}
  open={true}
>
  <WithStyles(ForwardRef(SnackbarContent))
    action={
      Array [
        <ForwardRef(WithStyles)
          aria-label="close"
          className="btn-msg-close"
          color="inherit"
          onClick={[Function]}
        >
          <Memo
            className="icon"
          />
        </ForwardRef(WithStyles)>,
      ]
    }
    aria-describedby="client-snackbar"
    className="error"
    message={
      <span
        className="message"
        id="client-snackbar"
      >
        <Memo
          className="icon icon-variant"
        />
      </span>
    }
  />
</WithStyles(ForwardRef(Snackbar))>
`;
